vars/cleanSctRunners.groovy

```diff
@@ -13,11 +15,19 @@ def call(Map params, RunWrapper currentBuild){
     env
 
     echo "Test status on runCleanupResource is: " + "$test_status"
+    echo "Params in runCleanupResource are: " + "$params"
+    echo "Collect logs status in runCleanupResource:" + "$collect_logs_status"
 
     echo "Starting to clean runner instances"
+    export RUNNER_IP=\$(cat sct_runner_ip||echo "")
+
+    if [ "$test_status" == "SUCCESS" ] || [ "$post_behaviour" == "destroy" ] && [ "$collect_logs_status" == "true" ]; then
+        ./docker/env/hydra.sh set-runner-tags --runner-ip \${RUNNER_IP} --tags keep 0
+    fi
+
     if [[ "$cloud_provider" == "aws" || "$cloud_provider" == "gce" || "$cloud_provider" == "azure" ]]; then
         export RUNNER_IP=\$(cat sct_runner_ip||echo "")
-        ./docker/env/hydra.sh clean-runner-instances --test-status "$test_status" --runner-ip \${RUNNER_IP}
+        ./docker/env/hydra.sh clean-runner-instances --test-status "$test_status" --runner-ip \${RUNNER_IP} --logs-collected "$collect_logs_status"
```

Comment:
what's `--logs-collected` mean in context of cleaning instances ?, it's not exactly clear

Reply:
As explained above: it's used by the cleanup logic for resetting the tag values based on if the test failed before it's target duration and if the logs were collected in an earlier stage.